api/src/main/java/io/strimzi/api/kafka/model/KafkaResources.java

```diff
@@ -30,6 +30,16 @@ public static String zookeeperPodName(String clusterName, int podNum) {
         return zookeeperStatefulSetName(clusterName) + "-" + podNum;
     }
 
+    /**
+     * Returns the name of the Zookeeper {@code Pod} for a {@code Kafka} cluster of the given name.
+     * @param ssName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @param podNum The number of the Zookeeper pod
+     * @return The name of the corresponding Zookeeper {@code Pod}.
+     */
+    public static String zookeeperPodNameSufix(String ssName, int podNum) {
```

Comment:
I think the name is misleading, since it's not returning a suffix – it's returning the full name. Something like `zookeeperPodNameFromStatefulSet()` might be better, but even then I'm not convinced this belongs on `KafkaResource`: This is in `api` remember, and it's much more common for those users to have to name of a Kafka cluster to hand, than the name of the SS.